nemo_text_processing/text_normalization/normalize.py

```diff
@@ -267,7 +271,7 @@ def select_verbalizer(self, lattice: 'pynini.FstLike') -> str:
 def parse_args():
     parser = ArgumentParser()
     parser.add_argument("input_string", help="input string", type=str)
-    parser.add_argument("--language", help="language", choices=["en"], default="en", type=str)
+    parser.add_argument("--language", help="language", choices=["en", "de", "ru"], default="en", type=str)
```

Comment:
normlize.py is deterministic=True, `de` and `ru` should be removed to avoid confusion

Reply:
done

Reply:
"de" and "ru" are still there

Reply:
its outdated file